File: tests/fixtures/clojure/basics.clj
(ns example.user
  (:require [clojure.string :as str]))

(defrecord User [id name email])

(defn greet [user]
  (str "Hello, " (:name user) "!"))

(def user (->User 1 "Alice" "alice@example.com"))
(println (greet user))

File: tests/fixtures/zig/basics.zig
const std = @import("std");

const User = struct {
    id: u32,
    name: []const u8,
    email: ?[]const u8,
};

pub fn main() !void {
    const user = User{
        .id = 1,
        .name = "Alice",
        .email = "alice@example.com",
    };
    std.debug.print("Hello, {s}!\n", .{user.name});
}